2025 Remake (#4)
endulum/dc-shuffle-clock: README.md

# Cave Shuffle Clock

A simple clock that sounds an alert whenever the current time approaches a minute multiple of five, for use with [Dragon Cave.](https://dragcave.net)

### About the tool

In Dragon Cave, eggs in each of the six biomes rotate out every five minutes on the dot. Being attentive of such rotations, or "shuffles" as they are referred to here, is key to effective cave hunting. The purpose of this tool is to keep the forgetful or multitasking user aware of such shuffles by way of a distinct auditory and visual cue, and to make looking for that one egg you want just a little less of a hassle.

### Features

- Runs perpetually in the background once started, until paused or closed
- Alert volume, sound choice, and timing are customizable
- Uses your local time and omits the hour
- Sends notifications (if supported) with an optional direct link to biome of choice
- Alert preferences are locally stored and remembered by the browser
- PWA-compatible and can be installed to your home screen

_This tool was coded with ToC compliance in mind. The page makes no requests to Dragon Cave and does not interact with the site in any way._

### Troubleshooting

> **Mobile users:** The Shuffle Clock is prone to be put to sleep by your mobile browser or device if not in focus, resulting in dropped alerts. The mobile environment is more aggressive about putting processes to sleep, and "forcing" the page to bypass mobile sleep is out of my control. If you're experiencing dropped alerts, ensure that you:

Reviewer: ```suggestion
**Mobile users:** Mobile browsers tend to sleep inactive pages when not in focus, resulting in dropped alerts. To minimise this, ensure that you:
```
